examples/dropwizard/pom.xml

```diff
@@ -0,0 +1,148 @@
+                                <args>
+                                    <arg>server</arg>
+                                    <arg>${dropwizard.server.config}</arg>
+                                </args>
+                                <ports>
+                                    <port>8080</port>
+                                </ports>
+
+                                <!-- good defaults intended for containers -->
+                                <jvmFlags>
+                                    <jmxFlag>-server</jmxFlag>
+                                    <jmxFlag>-Djava.awt.headless=true</jmxFlag>
+                                    <jmxFlag>-XX:+UnlockExperimentalVMOptions</jmxFlag>
+                                    <jmxFlag>-XX:+UseCGroupMemoryLimitForHeap</jmxFlag>
+                                    <jmxFlag>-XX:InitialRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:MinRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:MaxRAMFraction=2</jmxFlag>
+                                    <jmxFlag>-XX:+UseG1GC</jmxFlag>
+                                    <jmxFlag>-XX:MaxGCPauseMillis=100</jmxFlag>
+                                    <jmxFlag>-XX:+UseStringDeduplication</jmxFlag>
+                                </jvmFlags>
+                            </container>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>dockerBuild</id>
+                                <goals>
+                                    <goal>dockerBuild</goal>
+                                </goals>
+                                <phase>package</phase>
+                            </execution>
+                        </executions>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+    </profiles>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.dropwizard</groupId>
+                <artifactId>dropwizard-bom</artifactId>
+                <version>${dropwizard.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.dropwizard</groupId>
+            <artifactId>dropwizard-core</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>de.thomaskrille</groupId>
+            <artifactId>dropwizard-template-config</artifactId>
+            <version>${dropwizard-template-config.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>1.6.0</version>
+                <executions>
```

Comment:
I think the `<executions>` part is unnecessary.

Reply:
Remove the `exec:java` entirely, then?

Reply:
Oh, you are saying that `<id>java</id>. <goal>java</goal>` is redundant?

I can play around with it, if I understand the context is just to get `mvnw exec:java` working alongside the `jibDockerBuild`

Reply:
> Oh, you are saying that `<id>java</id>`. `<goal>java</goal>` is redundant?

Yeah, including the opening and closing `<executions>` part. From my testing, it works without them. You could also check the effective pom before and after removing the section (`mvnw help:effective-pom`) to see if there is any difference.

Reply:
An [`<execution>`](https://maven.apache.org/guides/mini/guide-configuring-plugins.html#Using_the_executions_Tag) is necessary to either (1) bind the the specific goals to a phase or (2) to be able to specialize the configuration by referencing the execution ID like `mvn exec:java@executionID`.  Maven will use an execution with an ID `default-cli` when invoked from the command-line.

Reply:
@briandealwis So, what are your recommendations here?

Reply:
My interpretation is that we should remove it, since we don't want to care about the execution ID here. (We won't do `mvnw exec:java@java`). Just verify that `mvnw exec:java` works great.